### PR TITLE
ci: pass test_type to ci_entry_point.sh

### PR DESCRIPTION
**Pull request by jongwu**
Pass test_type to ci_entry_point.sh to adjust test range of jenkins CI.

Fixes: #5339
Signed-off-by: Jianyong Wu <jianyong.wu@arm.com>

### CODE REVIEW

**Comment by jongwu** [comment]
Help TAL @GabyCT @jodh-intel , Thanks!

**Review by jodh-intel — APPROVED** [review]
Thanks @jongwu.

lgtm

File: .ci/ci_entry_point.sh [hunk 1]
```diff
@@ -37,6 +37,8 @@ repo_to_test="${1:-}"
 repo_to_test=${repo_to_test#"https://"}
 repo_to_test=${repo_to_test%".git"}
 
+# test type pass to jenkins_job_build.sh
+TEST_TYPE="${2:-all}"
```
Comment (littlejawa) [inline .ci/ci_entry_point.sh:41]: nit: maybe update the "usage" comment at the top, to include this optional parameter?

**Review by littlejawa — APPROVED** [review]
LGTM 
Thanks!

File: .ci/ci_entry_point.sh (hunk 1 above)
Comment (jongwu) [reply .ci/ci_entry_point.sh:41]: Good point! thanks.

**Pull request merged**